hockeysdk/src/main/java/net/hockeyapp/android/FeedbackActivity.java

```diff
@@ -230,9 +230,22 @@ public void onCreate(Bundle savedInstanceState) {
 
         initFeedbackHandler();
         initParseFeedbackHandler();
+        restoreSendFeedbackTask();
         configureAppropriateView();
     }
 
+    private void restoreSendFeedbackTask() {
+        Object object = getLastNonConfigurationInstance();
+        if (object != null && object instanceof SendFeedbackTask) {
+            mSendFeedbackTask = (SendFeedbackTask) (object);
```

Comment:
No. still need. I meant the parenthesis rounding `object` variable is redundant. 😄